go/client/cmd_simplefs_reset.go

```diff
@@ -31,6 +32,12 @@ func NewCmdSimpleFSReset(
 				Contextified: libkb.NewContextified(g)}, "reset", c)
 			cl.SetNoStandalone()
 		},
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "id",
+				Usage: "the ID to which to reset the folder (experts only!)",
```

Comment:
Maybe something even more explicit like "this can break this folder if you don't know exactly what you're doing"? or do you think this is discouraging enough?

Reply:
I think we try to keep the help length to under 80 chars, so I didn't want to do too much.  But also, most people aren't going to be able to figure out what to put in there -- it will have to parse as a real TLF ID, and also the TLF it corresponds to has to match the given handle, so it seems pretty safe to me as is.  The command itself is basically self-discouraging.